lib/modules/dosomething/dosomething_image/dosomething_image.module

```diff
@@ -165,6 +159,28 @@ function dosomething_image_get_themed_image_by_fid($fid, $style, $alt = '') {
 }
 
 /**
+ * Returns no image tag.
+ *
+ * Uses static cache to speed up the output.
+ *
+ * @return string
+ *   Img tag of no-image-found.png
+ */
+function dosomething_image_get_themed_no_image() {
+  $img = &drupal_static(__FUNCTION__);
```

Comment:
Actually, this is very common cache practice in drupal: https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_static/7
Plus, I mentioned this in the function docblock.

But you're right, I'll add the comment anyways.